src/components/viewers/MultiFlatmap.vue

```diff
@@ -270,7 +272,13 @@ export default {
           const markerIdentifier =
             store.state.settings.featuredMarkerIdentifiers[index];
           if (!markerIdentifier) {
-            this.addFeaturedMarker(markers[index], index, flatmap);
+            let markerExists = this.addFeaturedMarker(markers[index], index, flatmap);
```

Comment:
This is inside a for loop, better leave markerExists as true as long as one exists. Thanks.